modules/op-additional-options-webhooks.adoc

```diff
@@ -0,0 +1,150 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-additional-options-webhooks_{context}"]
+= Setting additional options for webhooks
+
+Optionally, you can set several additional options for the webhooks that several controllers in {pipelines-sshortname} create. The available options are `failurePolicy`, `timeoutSeconds`, and `sideEffects`. For information about these options, see the https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/[Kubernetes documentation].
+
+.Prerequisites
+
+* You installed the `oc` command-line utility
+
+* You are logged on to your {OCP} cluster with administrator rights for the namespace in which {pipelines-shortname} is installed, typically the `openshift-pipelines` namespace.
+
+.Procedure
+
+. View the list of webhooks that the {pipelines-shortname} controllers created. There are two types of webhooks: mutating webhooks and validating webhooks.
+.. To view the list of mutating webhooks, enter the following command:
++
+[source,terminal]
+----
+$ oc get MutatingWebhookConfiguration
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                             WEBHOOKS   AGE
+annotation.operator.tekton.dev   1          4m20s
+proxy.operator.tekton.dev        1          4m20s
+webhook.operator.tekton.dev      1          4m22s
+webhook.pipeline.tekton.dev      1          4m20s
+webhook.triggers.tekton.dev      1          3m50s
+----
+.. To view the list of validating webhooks, enter the following command:
++
+[source,terminal]
+----
+$ oc get ValidatingWebhookConfiguration
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                                 WEBHOOKS   AGE
+config.webhook.operator.tekton.dev                   1          4m24s
+config.webhook.pipeline.tekton.dev                   1          4m22s
+config.webhook.triggers.tekton.dev                   1          3m52s
+namespace.operator.tekton.dev                        1          4m22s
+validation.pipelinesascode.tekton.dev                1          2m49s
+validation.webhook.operator.tekton.dev               1          4m24s
+validation.webhook.pipeline.tekton.dev               1          4m22s
+validation.webhook.triggers.tekton.dev               1          3m52s
+----
+
+. In the `TektonConfig` custom resource (CR), add configuration for mutating and validating webhooks under the section for each of the controllers as necessary, as shown in the following examples. Use the `validation.webhook.pipeline.tekton.dev` spec for the validating webhooks and the `webhook.pipeline.tekton.dev` spec for the mutating webhooks.
++
+[IMPORTANT]
+====
+* You cannot set configuration for operator webhooks.
```

Comment:
Operator is always capitalized (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization) unless the use case here means something different than an actual OCP Operator.